packages/webapp/src/pages/Team/Billing/Show.tsx

```diff
@@ -3,62 +3,72 @@ import { useMemo } from 'react';
 import { Helmet } from 'react-helmet';
 import { Link } from 'react-router-dom';
 
+import { PaymentMethods } from './components/PaymentMethod';
+import { PlanCard } from './components/PlanCard';
 import { ErrorPageComponent } from '../../../components/ErrorComponent';
+import { Info } from '../../../components/Info';
 import { LeftNavBarItems } from '../../../components/LeftNavBar';
 import { Skeleton } from '../../../components/ui/Skeleton';
 import * as Table from '../../../components/ui/Table';
 import { Button } from '../../../components/ui/button/Button';
 import { useEnvironment } from '../../../hooks/useEnvironment';
 import { useApiGetPlans, useApiGetUsage } from '../../../hooks/usePlan';
+import { useStripePaymentMethods } from '../../../hooks/useStripe';
 import DashboardLayout from '../../../layout/DashboardLayout';
 import { useStore } from '../../../store';
-import { cn } from '../../../utils/utils';
+import { formatDateToInternationalFormat } from '../../../utils/utils';
 
+import type { PlanDefinitionList } from './types';
 import type { GetUsage, PlanDefinition } from '@nangohq/types';
 
-interface PlanDefinitionList {
-    plan: PlanDefinition;
-    active: boolean;
-    isDowngrade?: boolean;
-    isUpgrade?: boolean;
-}
-
 export const TeamBilling: React.FC = () => {
     const env = useStore((state) => state.env);
 
     const { error, plan: currentPlan, loading } = useEnvironment(env);
     const { data: plansList } = useApiGetPlans(env);
     const { data: usage, error: usageError, isLoading: usageIsLoading } = useApiGetUsage(env);
+    const { data: paymentMethods } = useStripePaymentMethods(env);
 
-    const plans = useMemo<PlanDefinitionList[]>(() => {
+    const list = useMemo<null | { list: PlanDefinitionList[]; activePlan: PlanDefinition }>(() => {
```

Comment:
I think `plans` makes more sense, specially when accessing:
`list.list`,  `list.activePlan`.
```suggestion
    const plans = useMemo<null | { list: PlanDefinitionList[]; activePlan: PlanDefinition }>(() => {
```

Reply:
true